aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -147,7 +148,17 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
     } catch (RuntimeException persistFailure) {
       LOG.error("Confirming if commit to {} indeed failed to persist, attempting to reconnect and check.",
```

Comment:
I think this logging should be moved around since `checkCommitStatus` is now conditional